### ISSUE TITLE
error on confirming location

### ISSUE THREAD

**Issue by danbjoseph** [JonathanGarro/SIMS-Portal#7]
searched a location, the map came up and it asked me to confirm, i did and got this error 
<img width="1959" alt="Screen Shot 2023-03-16 at 8 42 54 AM" src="https://user-images.githubusercontent.com/4806884/225620667-ac45e9e2-19b8-4187-94c2-0228867c9bb9.png">

**Comment by JonathanGarro** [comment]
yeah that route is doing two things: pinging the API to translate the location into coordinates and get the timezone, and update a json file to display a point on a map on the landing page. the latter is what's broken.
